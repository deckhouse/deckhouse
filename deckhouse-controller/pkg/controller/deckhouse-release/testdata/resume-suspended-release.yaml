apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.25.0
  annotations:
    release.deckhouse.io/suspended: "true"
spec:
  version: "v1.25.0"
status:
  phase: Suspended
