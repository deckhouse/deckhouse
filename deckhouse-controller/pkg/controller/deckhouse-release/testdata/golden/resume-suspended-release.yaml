---
apiVersion: deckhouse.io/v1alpha1
approved: true
kind: DeckhouseRelease
metadata:
  annotations:
    dryrun: "true"
    release.deckhouse.io/isUpdating: "false"
    release.deckhouse.io/notified: "false"
  creationTimestamp: null
  labels:
    heritage: deckhouse
  name: v1.15.0
  resourceVersion: "2"
spec:
  version: v1.15.0
status:
  approved: true
  message: Release object was restored
  phase: Deployed
  transitionTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  creationTimestamp: null
  name: v1.25.0
  resourceVersion: "1001"
spec:
  version: v1.25.0
status:
  approved: false
  message: ""
  phase: Pending
  transitionTime: null
